proof/archive.go

```diff
@@ -366,6 +579,12 @@ func (f *FileArchiver) FetchProofs(_ context.Context,
 				"unable to parse script key: %w", err)
 		}
 
+		outPoint, err := ParseOutPoint(parts[1])
```

Comment:
IIUC this would fail as we've written a truncated TXID, and the separator between the TXID and outpoint index is now `-` and not `:`.

Related, I don't think we can recover the proof locator from the proof path (as we do at the end of this loop), if we're truncating the TXID. One option is to use a larger alphabet like bech32 or base58, if we don't care about the paths matching the assetID and scriptkey displayed elsewhere.

Reply:
Yeah this should fail as the outpoint also won't be large enough.

Reply:
Yeah, I forgot to update this after deciding to truncate the outpoint. It's fixed now (not super nice as we need to parse the proof file to extract the full outpoint, but the `FetchProofs` method is only used in a single place when a re-org causes multiple files to be updated, so it shouldn't be too bad).